src/Driver/RemoteClient.php

```diff
@@ -605,7 +605,7 @@ private function respond(Request $request, string $buffer): \Generator
             $this->close();
             return;
         } catch (\Throwable $exception) {
-            $this->logger->error($exception);
+            $this->logger->error($exception->getMessage(), ['exception' => $exception]);
```

Comment:
```suggestion
            $this->logger->error('An exception was thrown while processing a request.', ['exception' => $exception]);
```

Reply:
I'm not really satisfied with this and the case in ExceptionMiddleware. I think the messages should be different.